clayui.com/content/docs/components/css-icons.mdx

```diff
@@ -15,6 +16,8 @@ description: 'Icons are a visual representation of an idea and/or action.'
 </div>
 </div>
 
+<IconSearch />
+
```

Comment:
hey @kresimir-coko and @bryceosterhaus I would add this component below the warnings, it is important that they are visible as soon as the user enters the page.